src/videojs-hls.js

```diff
@@ -950,6 +954,7 @@ videojs.Hls.prototype.drainBuffer = function(event) {
   // the playlist
   if (this.duration() !== Infinity && mediaIndex + 1 === playlist.segments.length) {
     this.mediaSource.endOfStream();
+    this.lastSeekedTime_ = undefined;
```

Comment:
In that case, I think we should be consistent about what value we choose for `lastSeekedTime_` to indicate "unset". What do you think about initializing it to `null` in the init function and then just checking for null?